pages/downloads/0.9.x-release-notes/[...slug].js

```diff
@@ -276,197 +240,42 @@ export default function PostPage({ frontmatter, content, id }) {
 
             <ReactMarkdown
```

Comment:
Instead of creating a whole new component as `RenderHeading`, can you re-use https://github.com/ballerina-platform/ballerina-dev-website/blob/master/components/common/main-content/MainContent.js.

Refer https://github.com/ballerina-platform/ballerina-dev-website/blob/master/pages/learn/integration/%5B...slug%5D.js

Reply:
After using that component I was getting some errors in swan-lake-release-notes rest all is working fine. I will try to fix it again